### ISSUE TITLE
[Bug]: After install windows version I canno use because there is a blocking error

### ISSUE THREAD

**Issue by giosueDelgado** [GetPublii/Publii#1309]
### Operating system

Windows 11

### Publii version

0.44.2 and 0.44.1

### Post editor

None

### Bug description

When I download the .exe file I cannot start publii:

![image](https://github.com/GetPublii/Publii/assets/11545723/f0ab71c8-8a71-4ef1-9e4e-fc8cdba71d12)

### Steps to reproduce

See description

### Relevant log output

```shell
See screenshot
```

**Comment by dziudek** [comment]
Hi, the folder with sites data is synced using One Drive?

**Comment by giosueDelgado** [comment]
This appen when I install the installer I not create any site or folders..

**Comment by dziudek** [comment]
Ok, I will ask in a different way - your documents folder where Publii files should be placed is synced with One Drive? :)

**Comment by giosueDelgado** [comment]
Yes my document folder is synced with One Drive

**Comment by dziudek** [comment]
Probably it is a reason of your issue - I will try to investigate such configuration on my windows laptop

**Comment by giosueDelgado** [comment]
Ok thank! I wait your updates

**Comment by giosueDelgado** [comment]
Note: If I close all processes for "Onedrive" the issue persists..

**Comment by dziudek** [comment]
I have checked this issue on my Windows laptop - I was only able to reproduce situation when I have freed space and put sites folder on One Drive - then the error about missing sites folder appeared. But I am unable to reproduce a problem after installing Publii - for me it always create folders properly - even with One Drive sync enabled.

Maybe you should try to mark the Publii folder as "Always keep on this device"? https://techcommunity.microsoft.com/t5/image/serverpage/image-id/231681iA6F0A120B0328EDF/image-dimensions/2500?v=v2&px=-1

**Comment by giosueDelgado** [comment]
Hello @dziudek, your suggestion did not work in my case.

But I decided to delete the Publii folder under my Documents and start again Publii, in this case, it starts from zero correctly.

So my issue can be closed.

Just a feedback: after this, I started to try to use this CMS for replace WordPress, but unfortunately I cannot use this for this missing basic feature: https://github.com/GetPublii/Publii/issues/60 so I will search for an alternative that supports multilanguage! 

Thanks for your support and time!
